### PR TITLE
Bellowshake support

### PR DESCRIPTION
**Pull request by 3araht**
## Description

Bellow Shake is supported.
(Layer change is used to change the Close layout and Open layout. Now it is possible to switch notes by toggling the layer when a foot switch is used.)

Pull request again (https://github.com/qmk/qmk_firmware/pull/20527)

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/bandominedoni/bandominedoni.h [hunk 1]
```diff
@@ -34,12 +34,169 @@
...
+
+#define _________________COLEMAK_R1________________ KC_J,    KC_L,    KC_U,    KC_Y,    KC_SCLN
+#define _________________COLEMAK_R2________________ KC_H,    KC_N,    KC_E,    KC_I,    KC_O
+#define _________________COLEMAK_R3________________ KC_K,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH
+
+#define _________________NUMBER_L__________________ KC_1,    KC_2,    KC_3,    KC_4,    KC_5
+// #define _________________NUMBER_R__________________ KC_6,    KC_7,    KC_8,    KC_9,    KC_0
+
+#define _________________FUNC__L___________________ KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5
+// #define _________________FUNC__R___________________ KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10
```
Comment (zvecr) [inline keyboards/bandominedoni/bandominedoni.h:61]: We wouldnt want this at the keyboard level.

File: keyboards/bandominedoni/bandominedoni.h [hunk 2]
```diff
@@ -34,12 +34,169 @@
...
+    MY_C = MY_TONE_MIN,
+    MY_Cs,
+    MY_Db = MY_Cs,
+    MY_D,
+    MY_Ds,
+    MY_Eb = MY_Ds,
+    MY_E,
+    MY_F,
+    MY_Fs,
+    MY_Gb = MY_Fs,
```
Comment (zvecr) [inline keyboards/bandominedoni/bandominedoni.h:88]: Also fairly sure we wont want this being copy pasted to multiple keyboards as it seems to be.

Also use of `QK_USER` at the keyboard level is incorrect. If this is trying to work around some limitation in core, then the fix should be targeted there instead.

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/bandominedoni/info.json [hunk 3]
```diff
@@ -3,20 +3,113 @@
     "manufacturer": "3araht",
     "url": "https://github.com/3araht",
     "maintainer": "3araht",
+    "features": {
+        "midi": true,
+        "extrakey": true,
+        "bootmagic": false,
+        "command": false,
+        "console": false,
+        "encoder_map": true,
```
Comment (zvecr) [inline keyboards/bandominedoni/info.json:12]: `encoder_map` is never allowed to be enabled at the keyboard level.

All `rules.mk` changes regarding `encoder_map` will also need to be reverted.
```suggestion
```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/bandominedoni/bandominedoni.h (hunk 2 above)
Comment (3araht) [reply keyboards/bandominedoni/bandominedoni.h]: Thank you for your advice. I will submit a pull request to expand the area for kb.
It used to be kb_user = 256_256, then reallocated to 64_448. (https://github.com/qmk/qmk_firmware/pull/19907)
I'll propose 128_384.

Comment (3araht) [reply keyboards/bandominedoni/bandominedoni.h]: Pull request has been issued here (https://github.com/qmk/qmk_firmware/pull/21950).

Comment (zvecr) [reply keyboards/bandominedoni/bandominedoni.h]: Making the range larger is not the correct change here. #20352 shows you are attempting to duplicate core logic, to fix an issue.  The actual issue with MIDI should be fixed, not this mess of code continually duplicated across multiple keyboards.

Comment (3araht) [reply keyboards/bandominedoni/bandominedoni.h]: Oh... I see. You're right.
I'll drop the pull request https://github.com/qmk/qmk_firmware/pull/21950

Then I guess the keyboard matrix location should be considered to differentiate which button was pressed.
I'm not sure if that's possible with my skill...

File: keyboards/bandominedoni/info.json
Comment (drashna) [inline keyboards/bandominedoni/info.json:1]: Please run `qmk format-json -i` on this file.

**Pull request closed without merging**